### PR TITLE
meta(codeowners): Transfer processing tasks

### PR DESCRIPTION
**Pull request by jjbayer**
Update codeowners to reflect new team responsibilities.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (jjbayer) [inline .github/CODEOWNERS:1]: This is just my best guess, feel free to disagree or extend the list.

**Review by olksdr — APPROVED** [review]

**Review by iker-barriocanal — APPROVED** [review]

File: .github/CODEOWNERS
Comment (Swatinem) [inline .github/CODEOWNERS:345]: 😱 🙀 
image

**Review by Swatinem — COMMENTED** [review]
I believe @ashwoods wanted to create a new team for this, I believe `owners-native` is double-deprecated by now :-D

File: .github/CODEOWNERS
Comment (jjbayer) [reply .github/CODEOWNERS:345]: Let me know if I went too far here. I was just making assumptions based on @ashwoods email.

Comment (ashwoods) [reply .github/CODEOWNERS:345]: We now have @getsentry/team-processing @getsentry/owners-processing. Re-processing is still an open topic, but I think we should own it instead of leaving it unowned, and expectations adjusted ;)

File: .github/CODEOWNERS
Comment (jjbayer) [inline .github/CODEOWNERS:346]: I don't know why github marks these rules as invalid. Am I missing a typo here? https://github.com/orgs/getsentry/teams/owners-processing

Comment (kamilogorek) [reply .github/CODEOWNERS:346]: I think that team has no write access updated for this repo, which is required for codeowners to work.

Comment (iker-barriocanal) [reply .github/CODEOWNERS:346]: I think so too. I see it's a top level team in `getsentry`, and I guess it should be under `Staff/Engineering/...`

Comment (ashwoods) [reply .github/CODEOWNERS:346]: Approval still pending. @mitsuhiko Is this something you can do? For both codeowners and team teams. 

owners-processing_Team_·_Edit_·_General

Comment (jjbayer) [reply .github/CODEOWNERS:346]: @ashwoods thank you, this looks green now. I won't merge the PR without approval from the processing team itself though. In case it looks good to you, could you please give it a stamp?

**Review by Swatinem — APPROVED** [review]

**Comment by iker-barriocanal** [comment]
Should we also add `src/sentry/lang/`? I assume now processing also owns these?

**Pull request merged**
